icu4c/source/tools/genrb/filterrb.h

```diff
@@ -0,0 +1,88 @@
+// © 2018 and later: Unicode, Inc. and others.
+// License & terms of use: http://www.unicode.org/copyright.html
+
+#ifndef __FILTERRB_H__
+#define __FILTERRB_H__
+
+#include <list>
+#include <string>
+#include <map>
+#include <ostream>
+
+#include "unicode/utypes.h"
+
+
+class ResKeyPath {
```

Comment:
A little bit of docs for the class, please.

Reply:
Done.